mne/commands/tests/test_commands.py

```diff
@@ -202,4 +203,37 @@ def test_watershed_bem():
         mne_watershed_bem.run()
 
 
+@slow_test
+@requires_mne
+@requires_freesurfer
+@sample.requires_sample_data
+def test_flash_bem():
```

Comment:
FYI @agramfort got to the bottom of this, `nose` was monkey-patching `sys.stdout` with a `StringIO` instance, so now we triage our use of `sys.stdout` in `run_subprocess` to compensate.